package.json

```diff
@@ -64,11 +65,15 @@
     "commander": "^2.9.0",
     "css-loader": "^0.28.0",
     "cssnano": "^3.10.0",
+    "enzyme": "^2.8.0",
     "express": "^4.14.1",
     "file-loader": "^0.11.1",
     "friendly-errors-webpack-plugin": "^1.6.1",
     "history": "^3.2.1",
+    "identity-obj-proxy": "^3.0.0",
     "isomorphic-style-loader": "^1.1.0",
+    "jest": "^19.0.2",
```

Comment:
dependencies binaries are included in `./node_modules/.bin` as well.
@LucasPerso @johangirod confirmation?